README.md

```diff
@@ -1,21 +1,82 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## scalar-admin client tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
+It will help to `PAUSE` and `UNPAUSE` the application without losing the transaction.
+
+### Usage
+
+You can manage the scalar-admin integrated applications in the following ways
+
+#### Install
+
+The release versions of `scalar-admin` fat jar can be downloaded from [releases](https://github.com/scalar-labs/scalar-admin/releases) page of Scalar Admin.
+
+#### Build
+
+In case you want to build `scalar-admin` fat jar from the source:
 
-To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows.
 ```console
-$ java -jar scalar-admin-<version>-all.jar
+$ ./gradlew shadowJar
 ```
 
-You can also build the tool as follows.
+* The built fat jar file is `build/libs/scalar-admin-<version>-all.jar`
+
+#### Container
+
+You can pull the docker image from [Scalar's container registry](https://github.com/orgs/scalar-labs/packages/container/package/scalar-admin).
+
+If you want to use the `scalar-admin` container, you can do it as follows.
 
 ```console
-$ ./gradlew installDist
-$ build/install/scalar-admin/bin/scalar-admin
+$ docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> <command_arguments>
 ```
 
+If you want to use the `scalar-admin` container on kubernetes, you can do it as follows.
+
+```console
+$ kubectl run <NAME> -it --image=ghcr.io/scalar-labs/scalar-admin:<version> --restart=Never --rm -- <command_arguments>
+```
+
+* Note that you can specify the same command arguments even if you use the fat jar or the container.
+The example commands in the next section are shown with a jar, but you can run the commands with the container in the same way by replacing
+`java -jar scalar-admin-<version>-all.jar` with `docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version>` or `kubectl run <NAME> -it --image=ghcr.io/scalar-labs/scalar-admin:<version> --restart=Never --rm --`
+
+In case you want to build the docker image from the source:
+
+```console
+$ ./gradlew docker
+```
+
+#### Run
```

Comment:
Remove this section and integrate the contents **nicely** into the `Use` section.

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.